ENTITY/ApplyForceToEntityCenterOfMass.md

```diff
@@ -5,17 +5,18 @@ ns: ENTITY
 
 ```c
 // 0x18FF00FC7EFF559E 0x28924E98
-void APPLY_FORCE_TO_ENTITY_CENTER_OF_MASS(Entity entity, int forceType, float x, float y, float z, BOOL p5, BOOL isDirectionRel, BOOL isForceRel, BOOL p8);
+void APPLY_FORCE_TO_ENTITY_CENTER_OF_MASS(Entity entity, int forceType, float x, float y, float z, cs_type(BOOL) int nComponent, BOOL bLocalForce, BOOL bScaleByMass, BOOL bApplyToChildren);
 ```
 
-## Parameters
-* **entity**: 
-* **forceType**: Refer to [APPLY_FORCE_TO_ENTITY](#_0xC5F68BE9613E2D18)
-* **x**: 
-* **y**: 
-* **z**: 
-* **p5**: 
-* **isDirectionRel**: 
-* **isForceRel**: 
-* **p8**: 
+Apply a force to an entity. More research/documentation on the gtaforums can be found [here](https://gtaforums.com/topic/885669-precisely-define-object-physics/) and [here](https://gtaforums.com/topic/887362-apply-forces-and-momentums-to-entityobject/).
```

Comment:
Seconding this, we shouldn't be pointing to external links at all, anything that requires research should be added to the fivem-docs.